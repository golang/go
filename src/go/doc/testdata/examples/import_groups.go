// Copyright 2022 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package foo_test

import (
	"fmt"
	"time"

	"golang.org/x/time/rate"
)

func Example() {
	fmt.Println("Hello, world!")
	// Output: Hello, world!
}

func ExampleLimiter() {
	// Uses fmt, time and rate.
	l := rate.NewLimiter(rate.Every(time.Second), 1)
	fmt.Println(l)
}
