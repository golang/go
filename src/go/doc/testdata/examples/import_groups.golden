-- .Play --
package main

import (
	"fmt"
)

func main() {
	fmt.Println("Hello, world!")
}
-- .Output --
Hello, world!
-- Limiter.Play --
package main

import (
	"fmt"
	"time"

	"golang.org/x/time/rate"
)

func main() {
	// Uses fmt, time and rate.
	l := rate.NewLimiter(rate.Every(time.Second), 1)
	fmt.Println(l)
}
